src/components/Tabs.tsx

```diff
@@ -0,0 +1,49 @@
+import classNames from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Tab } from '../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+}) => {
+  const { tabId } = useParams();
+  const selectedTab = tabs.find(tab => tab.id === tabId);
```

Comment:
this works but it would be easier if you wrote it like this, so in line #44 you wouldn't need to use `selectedTab.content`
```suggestion
  const selectedTabContent = tabs.find(tab => tab.id === tabId)?.content;
```